docs/_docs/05-configuration.md

```diff
@@ -424,6 +425,35 @@ comments:
     issue_term: "pathname"
 ```
 
+#### giscus comments
+
+To use giscus you will need to [install the app](https://github.com/apps/giscus) to your GitHub repository.
+
+The next step is to go to <https://giscus.app> and fill out the desired settings. This will generate a javascript, which will provide you with some of the settings you will need to configure things below.
```

Comment:
Fix captialization to stay consist with rest of theme docs, e.g. **JavaScript**. Might read better as "This will generate JavaScript" as "a JavaScript" sounds weird to me.